.gitignore

```diff
@@ -1,5 +1,6 @@
 # Xcode
 build/
+!taskcluster/ci/build
```

Comment:
We have the same rules in Fenix and other mobile projects. For instance: https://github.com/mozilla-mobile/fenix/blob/3e8836f71a96ffd17fa689877d4c621a6ca64c1b/.gitignore#L24-L25